interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/table-cell-overview/table-cell-overview.component.ts

```diff
@@ -146,8 +146,10 @@ export class TableCellOverviewComponent
 
     if (
       item.type !== ActivityTypeEnum.Transaction ||
-      item.attributes.financialServiceProviderName !==
-        FinancialServiceProviders.intersolveVoucherWhatsapp
+      (item.attributes.financialServiceProviderName !==
+        FinancialServiceProviders.intersolveVoucherWhatsapp &&
+        item.attributes.financialServiceProviderName !==
+          FinancialServiceProviders.intersolveVoucherPaper)
```

Comment:
Should use `FSPS_WITH_VOUCHER_SUPPORT` 

```suggestion
      !FSPS_WITH_VOUCHER_SUPPORT.includes(item.attributes.financialServiceProviderName)
```

Which makes me wonder also if the `isIntersolveVoucher` computed property in the same file has the same issue as the one you're fixing. We can look at this together if it helps.

Reply:
I think you're right, we also need to change that computation. Do I understand correctly that you are suggesting to have an enum with voucher_fsps somewhere? Maybe best if we briefly talk indeed!

Reply:
That enum already exists! You just need to import and update this and the other computed property. And then test to make sure both work with paper vouchers ofc.

Slip in my DMs if you wanna look at it together. Or if you'd like me to take over that is also fine!